editor/find_in_files.cpp

```diff
@@ -886,7 +951,7 @@ void FindInFilesPanel::apply_replaces_in_file(String fpath, const Vector<Result>
 }
 
 String FindInFilesPanel::get_replace_text() {
-	return _replace_line_edit->get_text().strip_edges();
```

Comment:
That bit was there since forever - now, replacement text is fully set (even with border spaces)